shoop/core/order_creator/_creator.py

```diff
@@ -220,9 +209,6 @@ def create_order(self, order_source):
 
         self._assign_code_usages(order_source, order)
 
-        order_creator_finished.send(
-            sender=type(self), order=order, source=order_source)
-
         order.save()
         self.process_order_after_lines(source=order_source, order=order)
```

Comment:
Yeah, as noted in https://github.com/shoopio/shoop/pull/466#discussion_r63204654 the signal is not triggered at same state as previously. It used to be triggered between _assign_code_usages and the second save, but after this change it is called after the second save and even after the process_order_after_lines hook. Anyway, as order_creator_finished signal is new in 4.0, this shouldn't matter for backward compatibility. I don't see any other reason neither. So this is fine. Actually this order seems even more logical as the original, i.e. triggering order_creator_finished being the last action in order creating.